src/terminal7.js

```diff
@@ -154,13 +156,13 @@ export class Terminal7 {
                 .addEventListener("click", () => this.toggleHelp())
         document.getElementById("help-button")
                 .addEventListener("click", () => this.toggleHelp())
-        document.getElementById("divide-h")
-                .addEventListener("click", () =>  {
-                    if (this.activeG && this.activeG.activeW.activeP.sy >= 0.04)
+        const dH = document.getElementById("divide-h")
+        const dV = document.getElementById("divide-v")
+        dH.addEventListener("click", () =>  {
+                    if (this.activeG && !dH.classList.contains("off"))
```

Comment:
cut gestures still work... You can copy the logic there or maybe test it in `Pane.split`?